lib/rpa_paycode.py

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# -*- mode: python3 -*-
+# This file (c) 2020 Jonald Fyookball
+# Part of the Electron Cash SPV Wallet
+# License: MIT
+'''
+This implements the functionality for RPA (Reusable Payment Address) aka Paycodes
+'''
+
+from . import rpaaddr
```

Comment:
Please import these in the canonical alphabetical order...

Reply:
done in 060ca8f